bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -44,6 +44,9 @@ BPF_ARRAY(cache_miss, u64, NUM_CPUS);
 // cpu freq counters
 BPF_ARRAY(cpu_freq_array, u32, NUM_CPUS);
 
+int sample_rate = 1000;
+int counter = 1000;
```

Comment:
Shouldn't it be:
```
int sample_rate = SAMPLE_RATE;
int counter = SAMPLE_RATE;
```

Reply:
libbpf is pre-compiled, we cannot use compilation flags. In this case, we have to global variable to set it. I opened #824 so I can use `InitGlobalVar` function

Reply:
Got it!
If we need to hard code now, let's use a smaller value.